### PR TITLE
Update givebutter serializer to include links & docs

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

File: app/serializers/api/v2/organization_serializer.rb [hunk 1]
```diff
@@ -8,18 +8,39 @@ def initialize(event:)
...
-        }
+        result = {}
+        result[:data] = data unless data.empty?
+        result[:links] = links unless links.empty?
+        result[:meta] = meta unless meta.empty?
+
+        result
       end
 
+      private
```
Comment (garyhtou) [inline app/serializers/api/v2/organization_serializer.rb]: is this okay to be private if `::Api::V2::OrganizationsSerializer` needs to call it?

https://github.com/hackclub/bank/blob/3d649d58c32449e2524197813a1044c43ff9871f/app/serializers/api/v2/organizations_serializer.rb#L19-L19

File: app/serializers/api/v2/organization_serializer.rb [hunk 2]
```diff
@@ -8,18 +8,39 @@ def initialize(event:)
...
+        if @event.partnered_signup.present?
+          result[:sup] = Rails.application.routes.url_helpers.api_v2_partnered_signup_url(public_id: @event.partnered_signup.public_id)
+        end
+
+        result
+      end
+
+      def meta
+        @meta ||= {
+          docs: 'https://bank.hackclub.com/docs/api#/Organizations/v2Organizations'
```
Comment (garyhtou) [inline app/serializers/api/v2/organization_serializer.rb]: good idea!

File: app/serializers/api/v2/partnered_signup_serializer.rb [hunk 3]
```diff
@@ -8,25 +8,49 @@ def initialize(partnered_signup:)
...
-          country: @partnered_signup.country,
           organization_name: @partnered_signup.organization_name,
           organization_id: @partnered_signup.event&.public_id, # nil if event does not exist yet
+
+          ## (@msw) These fields are intentionally not public– this info should not leave bank
+          # owner_name: @partnered_signup.owner_name,
+          # owner_phone: @partnered_signup.owner_phone,
+          # owner_address: @partnered_signup.owner_address,
+          # owner_birthdate: @partnered_signup.owner_birthdate,
+          # country: @partnered_signup.country,
```
Comment (garyhtou) [inline app/serializers/api/v2/partnered_signup_serializer.rb]: :+1:  yeah makes sense

File: app/serializers/api/v2/organization_serializer.rb (hunk 1 above)
Comment (maxwofford) [reply app/serializers/api/v2/organization_serializer.rb]: Oh, I missed this! Go ahead and take it out

File: app/serializers/api/v2/partnered_signup_serializer.rb [hunk 4]
```diff
@@ -8,25 +8,49 @@ def initialize(partnered_signup:)
...
-        }
+        result = {}
+        result[:data] = data unless data.empty?
+        result[:links] = links unless links.empty?
+        result[:meta] = meta unless meta.empty?
+
+        result
       end
 
+      private
```
Comment (garyhtou) [inline app/serializers/api/v2/partnered_signup_serializer.rb]:
```suggestion
```

File: app/serializers/api/v2/organization_serializer.rb (hunk 1 above)
Comment (garyhtou) [reply app/serializers/api/v2/organization_serializer.rb]:
```suggestion
```

**Review by garyhtou — CHANGES_REQUESTED** [review]

**Review by garyhtou — APPROVED** [review]

**Pull request merged**